packages/office-ui-fabric-react/src/components/Rating/Rating.base.tsx

```diff
@@ -97,32 +97,34 @@ export class RatingBase extends BaseComponent<IRatingProps, IRatingState> {
     });
 
     for (let i = min as number; i <= (max as number); i++) {
-      const ratingStarProps: IRatingStarProps = {
-        fillPercentage: this._getFillingPercentage(i),
-        disabled: disabled ? true : false,
-        classNames: this._classNames
-      };
-
-      starIds.push(this._getStarId(i - 1));
-
-      stars.push(
-        <button
-          className={ css(this._classNames.ratingButton, {
-            [this._classNames.rootIsLarge]: size === RatingSize.Large,
-            [this._classNames.rootIsSmall]: size !== RatingSize.Large
-          }) }
-          id={ starIds[i - 1] }
-          key={ i }
-          { ...((i === Math.ceil(this.state.rating as number)) ? { 'data-is-current': true } : {}) }
-          onFocus={ this._onFocus.bind(this, i) }
-          disabled={ disabled || readOnly ? true : false }
-          role='presentation'
-          type='button'
-        >
-          { this._getLabel(i) }
-          <RatingStar key={ i + 'rating' }  {...ratingStarProps} />
-        </button>
-      );
+      if (i !== 0) {
```

Comment:
Technically this is actually wrong. if min is 3 and max is 5, we should render 2 stars where 3 represents none.

A better question might be, why on earth do we have "min"??? That seems silly.

Reply:
@cschleiden could probably answer more about the build. From what I can tell rating was built for a specific interaction and that was to click a star to set how many stars you want - like the [demo](https://developer.microsoft.com/en-us/fabric#/components/rating).  When the UI is set up this way, there is no 'no stars' selection - or I have seen them do that before where '0' is represented by a 'no entry sign' symbol at the left end.

Admittedly it makes little sense over 1, but the min by default is 1 because you can't select zero. Maybe it would be better to deprecate 'min' and have an 'allowNoStars' prop instead?

Reply:
I have really no idea why I added min. Seemed like a good idea at the time ;) Feel free to remove it.

Reply:
@cschleiden I thought maybe it could have been a way to create a readonly Rating. Like min/max = value. But that's all I could come up with! :-D

I think it would make sense to deprecate.

Reply:
@oengusmacinog minor ask; can you add a screener test to validate rendering value 0, 1, max? Would be nice to have this locked down but clearly we were missing a case.